README.md

```diff
@@ -41,59 +41,83 @@ These [values are what we value](http://www.slideshare.net/reed2001/culture-1798
 The initial list of these values were created at our first company retreat, when Bonusly was a team of seven people.
 
 * [Go get results](https://github.com/bonusly/un-handbook#go-get-results)
-* [Communicate](https://github.com/bonusly/un-handbook#communicate)
+* [Be an excellent teammate](https://github.com/bonusly/un-handbook#be-an-excellent-teammate)
 * [Default to transparency](https://github.com/bonusly/un-handbook#default-to-transparency)
-* [Work smarter, not harder](https://github.com/bonusly/un-handbook#work-smarter-not-harder)
-* [Help out](https://github.com/bonusly/un-handbook#help-out)
+* [Work smarter](https://github.com/bonusly/un-handbook#work-smarter)
 * [Seek and embrace diversity](https://github.com/bonusly/un-handbook#seek-and-embrace-diversity)
 * [Delight the customer](https://github.com/bonusly/un-handbook#delight-the-customer)
 
 ### Go get results
+* Bias toward action
+* Try things that might not work
+* Focus on results, not inputs
+* Take responsibility for understanding context and goals
+
 You can think of this value in two parts: “go get” and “results”. The first part speaks to our strong bias toward action. If there’s something you want to make, improve, change, or get rid of -- go do it. Need input or feedback? Chat with some colleagues on Slack. Need copywriting, engineering or design resources? Recruit those colleagues by rallying them to your cause. Whatever you do, avoid what [Netflix calls](http://www.slideshare.net/reed2001/culture-1798664/11-11_Impact_You_accomplish_amazing) “analysis paralysis”.
 
 The second part (“results”) speaks to the primacy of outputs. We don’t particularly value inputs - how the work gets done. What matters is getting the work done and measuring the impact of this work after completing it. Do the results match up with your goals? Is there more room for improvement? Were there unintended consequences (good or bad)?
 
-Share the results from your work (see our values of “Communicate” and “Default to Transparency”): if they’re great, you’ll be swimming in bonuses. If they weren’t what you hoped for, no big deal - get some input from the team and then keep executing until the results are great.
-
-### Communicate
-We hire team members who we believe will make us smarter and better at what we're trying to do. In order to succeed at becoming smarter and better as a whole, it's essential for us to communicate.
+In practice, this is challenging. In order to go get results, we must understand current goals and priorities, and cost/benefit tradeoffs (ie. go for the quick win that costs little, or spend more time on a robust solution?). Our core value of "Default to Transparency" should ensure that the information you need is available to you, but if it isn't, go seek it out.
 
```

Comment:
Add in a sentence about why this value is important to us? I really liked that in the next value description.